### PR TITLE
mosquitto: Adding some updates and corrections

### PR DESCRIPTION
**Pull request by munoz0raul**
Mosquitto Tutorial was broken because the files from extra-container weren't accepted on the pull request.

I updated the extra container and sent a new pull request there as well.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for ed8d19ba2de69446c0ffe911f5889354da03e461 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/621/docs/artifacts/html/index.html

File: source/tutorials/mosquitto.rst [hunk 1]
```diff
@@ -73,40 +73,31 @@ Download your factory container.git repository and follow the commands to copy t
...
-       - "1883:1883/tcp"
-     restart: always
-     read_only: true
+ $ tree mosquitto/
+ mosquitto/
+ ├── docker-compose.yml
+ ├── mosquitto.conf
+ └── run.sh
+
+The only files necessary to build the docker composer app are ``docker-compose.yml`` and ``run.sh``. The ``mosquitto.conf`` file will be used next to change the default mosquitto configuration.
```
Comment (doanac) [inline source/tutorials/mosquitto.rst:85]: I think mosquitto.conf is necessary. This is the default file used by the container?

Comment (doanac) [reply source/tutorials/mosquitto.rst:85]: I also see this line above is a little off:
~~~
$ git clone https://source.foundries.io/factories/getstartedvideo/containers.git/
~~~

You might do:
~~~
$ git clone https://source.foundries.io/factories/<your factory>/containers.git
~~~

to help prevent copy/paste bugs

File: source/tutorials/mosquitto.rst [hunk 2]
```diff
@@ -73,40 +73,31 @@ Download your factory container.git repository and follow the commands to copy t
...
 
 Add the changes to your Factory and wait for it to finish compiling your app::
 
  # Ubuntu Host Machine
- $ git add mosquitto/
- $ git commit "Adding new mosquitto app"
+ $ git add mosquitto/docker-compose.yml
+ $ git add mosquitto/run.sh
```
Comment (doanac) [inline source/tutorials/mosquitto.rst:91]: Can do `git add mosquitto/*`. I think you need mosquitto.conf

**Review by doanac — APPROVED** [review]

**Review by mike-scott — APPROVED** [review]
LGTM

**Pull request merged**